### DISCUSSION TITLE
How can I export a flightplan as just a set of coordinates?

### DISCUSSION THREAD

**Discussion by AlexSkeleton12** [albar965/littlenavmap#1166] [Q&A]
Hello,
I have this SID ![here](https://github.com/albar965/littlenavmap/assets/50224102/9ba47c62-6d07-446e-b053-4cb7943a3325) that I would like to export as a set of coordinates with as much accuracy as possible to replicate the curved shape of the turn. How can I do this (if I can)?
Thanks.

**Comment by albar965** [comment]
Enable all in menu `File` -> `Export Options` but take note of the limitations. LNM will not rebuild the bow using waypoints when exporting. Only waypoints will be saved.

**Reply by AlexSkeleton12** [reply]
Will it export the coordinates throughout the turn from TRN that is shown in the procedure?

**Reply by albar965** [reply]
Nope. Only waypoints.

**Reply by AlexSkeleton12** [reply]
Thanks for your answers and your great program. On another note, where are the SIDs and STARs stored in the sqlite file?

**Reply by albar965** [reply]
> Thanks for your answers and your great program. On another note, where are the SIDs and STARs stored in the sqlite file?

You're welcome. :slightly_smiling_face: 

SID and STAR are stored in the table `approach` (wrong name due to historical reasons). See comment on top of the table for SID and STAR encoding in this table: https://github.com/albar965/atools/blob/92509ece54e7b9cb13a56ada46344e6db858aca2/resources/sql/fs/db/create_ap_schema.sql#L419-L467

Alex
